pkg/ocm/ocmWrapper.go

```diff
@@ -45,9 +46,26 @@ type DefaultOCMInterfaceImpl struct{}
 
 var DefaultOCMInterface OCMInterface = &DefaultOCMInterfaceImpl{}
 
+func (o *DefaultOCMInterfaceImpl) SetupOCMConnection() (*ocmsdk.Connection, error) {
+	ocmNotLoggedInMessage := "Not logged in"
+
+	// Setup connection at the first try
+	connection, err := ocm.NewConnection().Build()
+	if err != nil {
+		if strings.Contains(err.Error(), ocmNotLoggedInMessage) {
+			return nil, fmt.Errorf("please make sure you have logged into OCM, " +
+				"use \"ocm login --use-auth-code --url $ENV\" to login ")
```

Comment:
Just a nit:

```suggestion
    return nil, fmt.Errorf("Failed to create OCM connection: please ensure you are logged into OCM using the command \"ocm login --use-auth-code --url $ENV\"")
```

Reply:
The "Failed to create OCM connection:" will be included in the OCM side error message, I just update the second part

Reply:
All good then.